src/Azure-Migrate-Export.csproj

```diff
@@ -157,8 +159,7 @@
     <Compile Include="Models\Assessment\Excel\CoreReport\SQL_All_Instances.cs" />
     <Compile Include="Models\Assessment\Excel\CoreReport\WebApp_PaaS.cs" />
     <Compile Include="Models\Assessment\Excel\CoreReport\Decommissioned_Machines.cs" />
-    <Compile Include="Models\Assessment\Excel\OpportunityReport\AVS_Summary.cs" />
-    <Compile Include="Models\Assessment\Excel\OpportunityReport\AVS_IaaS_Rehost_Perf.cs" />
+    <Compile Include="Models\Assessment\Excel\CoreReport\AVS_IaaS_Rehost_Perf.cs" />

```

Comment:
Why was this change required?

Reply:
I did this to shift all AVS excel worksheet related files to Core Report folder as mentioned in the previous comment: "Please ensure similar cases are handled everywhere.".